lib/alaveteli_text_masker.rb

```diff
@@ -47,10 +49,22 @@ def apply_masks(text, content_type, options = {})
   private
 
   def uncompress_pdf(text)
-    AlaveteliExternalCommand.run("pdftk", "-", "output", "-", "uncompress", :stdin_string => text)
+    temp = Tempfile.new(encoding: 'ascii-8bit')
```

Comment:
Perhaps these ought to get created within the app's `tmp` dir (with a name prefix), so that it's clear what they relate to:

```diff
- temp = Tempfile.new(encoding: 'ascii-8bit')
+ temp = Tempfile.new('pdftk', './tmp', encoding: 'ascii-8bit')
```